docs/content/references/cli/client.mdx

```diff
@@ -127,7 +127,7 @@ $ sui client objects 0x36df11369cf00ecf0be68d6ba965b0abe2e883bc5245911e3a29ebfa0
 
 ### Get complete object information
 
-Use `sui client object <OBJECT-ID>` to output complete details for the object with the ID you provide. This example checks the details for a Coin object. 
+Use `sui client object <OBJECT-ID>` to list an object's information by the ID you provide. In this example a Coin object's information is displayed.
```

Comment:
```suggestion
Use `sui client object <OBJECT-ID>` to list object information for the ID you provide. This example displays the information for a Coin object.
```

Reply:
Switching back to active voice on that last sentence. And avoid using possessive for inanimate objects. In other words, "to list object information" is purposively suggested instead of "to list an object's information"